cypress/page/shipping-step.page.ts

```diff
@@ -0,0 +1,19 @@
+class ShippingStepPage {
+    private checkbtn: string;
+    private termsCheck: string;
+
+    constructor() {
+        this.checkbtn = ".cart_navigation > .button > span";
+        this.termsCheck = "#cgv";
+    }
+
+    public check(): void {
+        cy.get(this.termsCheck).click();
+    }
+
+    public pcheckout(): void {
```

Comment:
I would recommend to change to
```suggestion
    public proceedCheckout(): void {
```
Because it is not intuitive what the "p" refers to, so this way it could improve readability